spec/rubocop/cop/lint/duplicate_methods_spec.rb

```diff
@@ -233,16 +233,14 @@ def self.another
       RUBY
     end
 
-    it 'registers an offense when class << exp is used' do
-      pending
-      expect_offense(<<~RUBY, 'test.rb')
+    it 'does not register an offense when class << exp is used' do
+      expect_no_offenses(<<~RUBY, 'test.rb')
         #{opening_line}
           class << blah
             def some_method
               implement 1
             end
             def some_method
-            ^^^^^^^^^^^^^^^ Method `A#some_method` is defined at both test.rb:3 and test.rb:6.
               implement 2
             end
           end
```

Comment:
This test will not be able to ever pass, because `Node#parent_module_name` will return `nil` because it cannot evaluate `class << blah`, so I have removed the `pending` and marked it as `expect_no_offenses`.

Reply:
I would mark it as `pending`, as ideally the cop would detect it.

Reply:
Ok I'll drop that commit then.